docs/languageversion2.md

```diff
@@ -0,0 +1,11 @@
+# Language Version 2.0 Triggers
+
+We have recently developed a new Language Version 2.0, which provides a new format for resource definitions when transpiling to an ARM Template.
+
+However, with this new language version, we noticed that it may cause breaking changes in some use cases.
```

Comment:
As a customer, this sentence wouldn't give me a lot of confidence in our engineering quality bar. "may cause breaking changes" doesn't give me any ability to evaluate risk when deciding to adopt the feature, and sounds like something I'd need to be very concerned about.

Could we make this more precise (e.g. "we are aware of the following behavioral differences", followed by a list with links to issues)?